app/views/choice_page.scala.html

```diff
@@ -22,48 +22,39 @@
 @import views.helpers.Title
 @import views.helpers.ErrorMapper.radioGroupErrors
 @import views.html.components.gds._
+@import controllers.routes.{InboxChoiceController, MrnEntryController}
 
 @this(
   mainTemplate: gdsMainTemplate,
   govukRadios: GovukRadios,
   govukButton: GovukButton,
   errorSummary: errorSummary,
+  paragraph: paragraph,
+  paragraphBody: paragraphBody,
+  pageTitle: pageTitle,
+  link: link,
   formHelper: FormWithCSRF
```

Comment:
It looks like you do not use anymore `govukRadios`, `govukButton`, `errorSummary` and `formHelper` so I suppose they can be removed.

Reply:
Good spot. Removed :+1: